jormungandr-lib/src/interfaces/config/node.rs

```diff
@@ -175,8 +175,7 @@ pub struct TrustedPeer {
 
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct NodeConfig {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub storage: Option<PathBuf>,
```

Comment:
This is a breaking change btw. Needs to label it as such and comment that this is going to change (cannot *not* have a storage)